source/docs/articles/sites/code/what-is-the-pantheon_api-module.md

```diff
@@ -14,6 +14,7 @@ The code for the API module is available within our upstream repositories:
 
 - [Drupal 6](https://github.com/pantheon-systems/drops-6/tree/master/modules/pantheon)
 - [Drupal 7](https://github.com/pantheon-systems/drops-7/tree/master/modules/pantheon)
+- [Drupal 8](https://github.com/pantheon-systems/drops-8/tree/master/modules/pantheon)
 
```

Comment:
Right. This module doesn't exist, and from the issue above, it looks like it might not exist. I think this PR should be revised when we understand that issue.